cpp/src/umap/runner.cuh

```diff
@@ -449,35 +547,16 @@ void _transform(const raft::handle_t& handle,
     n_epochs /= 3;
   }
 
-  CUML_LOG_DEBUG("n_epochs=%d", n_epochs);
-
-  raft::linalg::unaryOp<value_t>(
-    graph_coo.vals(),
-    graph_coo.vals(),
-    graph_coo.nnz,
-    [=] __device__(value_t input) {
-      if (input < (max / float(n_epochs)))
-        return 0.0f;
-      else
-        return input;
-    },
-    stream);
-
-  RAFT_CUDA_TRY(cudaPeekAtLastError());
-
-  /**
-   * Remove zeros
-   */
-  raft::sparse::COO<value_t> comp_coo(stream);
-  raft::sparse::op::coo_remove_zeros<value_t>(&graph_coo, &comp_coo, stream);
+  raft::sparse::COO<value_t> trimmed_graph(stream);
+  trim_graph(handle, graph_coo, trimmed_graph, inputs.n, n_epochs);
```

Comment:
I think here we would want to use the `n_epochs` set in the lines above for determining the threshold, but calling `traim_graph` will figure out a new `n_epoch` for the threshold inside the function.